### ISSUE TITLE
Disable MCU selection box if BIN file is loaded

### ISSUE THREAD

**Issue by nooges** [qmk/qmk_toolbox#187]
## Feature Request Type

- [ ] Core functionality
- [x] Alteration (enhancement/optimization) of existing feature(s)
- [ ] New behavior

## Description

For people flashing to non-AVR devices, the MCU dropdown can still be confusing, even though it says `(AVR only)`. Might be a good idea to disable the box if a .bin file is loaded in or do some other detection method if someone has a better idea.

**Comment by fauxpark** [comment]
I think we will need to refactor the bootloader detection process so that it only picks up the first device it finds. Then depending on which bootloader that is, it will be much simpler to set the state of the dropdown.

**Comment by nooges** [comment]
Yeah, I'd prefer that, my other thought along the same lines had been to check the `canFlash` status.

**Comment by drashna** [comment]
What about the tada, that uses a bin?

**Comment by yanfali** [comment]
> What about the tada, that uses a bin?

tada doesn't use toolbox, it uses file manager

**Comment by skullydazed** [comment]
I feel like it should be disabled until it's needed. It's pretty confusing and only actually needed in a limited number of circumstances.
